lib/wor/batchifier.rb

```diff
@@ -12,16 +12,24 @@ module Batchifier
     # => optionaly a batch_size and a merge strategy. It will slice the collection and
     # => apply the chozen strategy to all chunks and merge the results. It expects the responses
     # => to be Hash. It can ignore them if the given strategy is no_response
-    def execute_in_batches(collection, batch_size: 100, strategy: :add)
+    def execute_in_batches(collection, batch_size: 100, strategy: :add, &block)
       strategy_class = classify_strategy(strategy)
       collection.lazy.each_slice(batch_size).inject(strategy_class.new.base_case) do |rec, chunk|
-        response = yield(chunk)
+        response = batch_process(chunk, &block) if block_given?
         merge(response, rec, strategy)
       end
     end
 
     private
 
+    def batch_process(chunk)
+      yield chunk
+    rescue StandardError => e
+      puts "Backtrace:\n\t#{e.backtrace.join("\n\t")}"
```

Comment:
esto debería integrarse con un logger, preferiblemente

Reply:
That's a valid suggestion. What you think about something like this?

https://stackoverflow.com/questions/43348556/how-can-i-nicely-print-a-formatted-exception-that-i-catch-in-a-rescue-block/43348709